script.js

```diff
@@ -18,7 +18,11 @@ function editTask(pen){
         edit.style.display = "none";
     }
 }
-
+function trashTask(trashbin){
+    const id = trashbin.parentElement.parentElement.id;
+    const trash = document.querySelector(`#${id}`);
```

Comment:
You don't need the id for this task.
Skip the id of line 22 and call the variable of the line direct trash.